drivers/extcon/extcon-intel-mrfld.c

```diff
@@ -243,6 +244,8 @@ static int mrfld_extcon_probe(struct platform_device *pdev)
 
 	/* Get initial state */
 	mrfld_extcon_role_detect(data);
+	regmap_read(regmap, BCOVE_SCHGRIRQ1, &status);
```

Comment:
Add a blank line here and a comment about what you are doing, e.g. "Cached status value is used for cable detection, see comments in mrfld_extcon_cable_detect(), we need to sync cached value with a real state of the hardware".